main.go

```diff
@@ -22,7 +22,8 @@ import (
 var templates = template.Must(template.ParseGlob("templates/*.html"))
 
 func init() {
-    http.HandleFunc("/", testHandler)
+    http.HandleFunc("/tasks/populate-dev-data", populateDevData)
```

Comment:
`populateDevData` is for boostrapping if you're running the app for the first time in development.